packages/airnode-deployer/package.json

```diff
@@ -47,6 +48,7 @@
     "ts-node": "^10.1.0",
     "typescript": "^4.2.4",
     "webpack": "^5.51.1",
-    "webpack-cli": "^4.8.0"
+    "webpack-cli": "^4.8.0",
+    "webpack-ignore-dynamic-require": "^1.0.0"
```

Comment:
Not super excited to add this package with ~1000 weekly downloads and 5 total commits 😬

Reply:
I usually don't like it either but this is just a packaged [StackOverflow answer](https://stackoverflow.com/questions/38697321/webpack-require-expression-external) and it's only a few lines. I just used a package instead of copying those.